Personal stop words em .txt separado

matheusvvb-19 opened the pull request:
#12 
As palavras da lista _personal_stop_words_  agora estão em um arquivo _.txt_ separado.
Além disso, a obtenção dessas palavras foi feita em uma função - utilizada em _clean_text.py_ e _crawler.py_.

Commit:
personal stop words from txt file

matheusvvb-19 committed:
Update crawler.py

brenolf commented on crawler.py:
```diff
@@ -4,13 +4,17 @@
+    with open (file_path, 'rt') as file:
+      for line in file:
+        strings_list.append(line.rstrip('\n'))
+    return strings_list
```
💅 daria pra importar a função do outro arquivo, imagino?

matheusvvb-19 committed:
Merge branch 'priscilaportela:master' into dataCleaning

matheusvvb-19 committed:
import function form crawler

importando a função list_from_txt do arquivo crawler.py, visto que a mesma função é utilizada primeiramente lá.
